### PR TITLE
Boneskinner combat replay

### PR DESCRIPTION
**Pull request by Linkaaaaa**
- Added decorations for Death Wind, Crushing Cruelty, Douse in Darkness, Cascade and Grasp
- Fixed a couple of typos
- Added a missing image for 100cm
- For Cascade, i have approssimated the aoe size based on this video https://youtu.be/os49oKKG0X8 and corrisponding log [20230531-160923.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/11618866/20230531-160923.zip) if you wanna take a look at the measurements
- The combay replay map is currently not scaled correctly and a bit squished (also for kodans and fraenir i believe), linus will take care of it in the next few days and re-draw it.

### CODE REVIEW

**Comment by EliphasNUIT** [comment]
Re draw the maps? It isn't just a coordinate definition problem in GetCombatReplayMap?

**Comment by Linkaaaaa** [comment]
He wants to remove the yellow dots since there are torch gadgets displayed now and he doesn't have the original project anymore :D

File: GW2EIEvtcParser/EncounterLogic/Strikes/IcebroodSaga/Bjora/Boneskinner.cs [hunk 1]
```diff
@@ -99,5 +100,137 @@ protected override void SetInstanceBuffs(ParsedEvtcLog log)
...
+        {
+            if (log.CombatData.TryGetEffectEventsByGUID(guid, out IReadOnlyList<EffectEvent> rectangularIndicators))
+            {
+                foreach (EffectEvent indicator in rectangularIndicators)
+                {
+                    int duration = 360;
+                    int start = (int)indicator.Time;
+                    int end = (int)indicator.Time + duration;
+
+                    ParametricPoint3D point = target.GetCombatReplayPolledRotations(log).Where(x => x.Time > start && x.Time < end).FirstOrDefault();
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EncounterLogic/Strikes/IcebroodSaga/Bjora/Boneskinner.cs:223]: replace with replay.PolledRotations.Where(...)

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**